### ISSUE TITLE
bash-forwarder-template should define the same header as bash-template

### ISSUE THREAD

**Issue by Latankar** [sbt/sbt-native-packager#921]
both templates define different headers

bash-template:
```
#!/usr/bin/env bash
```
bash-forwarder-template:
```
#!/bin/sh
```

Executing a bash-forwarder-template based script (example-app) the following message is displayed:
```
./example-app: 21: [: example-app: unexpected operator
./example-app: 29: ./incoming-messages-processor-app: [[: not found
```

**Comment by muuki88** [comment]
Thanks for the report. Would you mind sending a pull request to fix this? I'm not sure why none of the existing tests in `src/sbt-test/bash` found this.

**Comment by jan0sch** [comment]
I guess that on most linux systems you won't notice because they usually link /bin/sh to bash.

**Comment by muuki88** [comment]
the `/bin/sh` is on purpose as this more general, I was wondering about `#!/usr/bin/env bash`

**Comment by jan0sch** [comment]
If `/bin/sh` is used then the script must be sh compatible which is often not the case because of the mentioned fact that most linux distributions replace sh with bash. This means that such errors will only trigger on systems that don't replace sh with bash (bsds for example).
Using `/usr/bin/env bash` requires that bash is installed  but provides a cross plattform way to execute them.
I don't know if a sh script will run on a bash though.

**Comment by jan0sch** [comment]
According to debian (https://wiki.debian.org/Shell) they used to use `bash` as `bin/sh` replacement but switched to `dash` as `bin/sh` replacement from Squeeze on forward.
I haven't researched this in depth for other distributions but if we consider the fact that most distributions replace `bin/sh` with something to their liking then it may be the safer bet to use `#!/usr/bin/env bash` in general. This explicitely requires a bash on the system and will circumvent any possible bugs that might occur if `bin/sh` has been replaced with some esoteric shell we might not even have heard about. ;-)

Shall I provide a PR that adjusts the forwarder-template to the env solution?

**Comment by muuki88** [comment]
Thanks for the research 👍 I found the pull request were a similar fix was introduced (#167). 

> then it may be the safer bet to use #!/usr/bin/env bash

Sounds good

> Shall I provide a PR that adjusts the forwarder-template to the env solution?

That would be awesome :)
